Dokan/Dokan/Scenes/ProductDetails/CollectionViewDelegate.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TableViewDelegate.swift
+//  Dokan
+//
+//  Created by heba isaa on 31/07/2022.
+//
+
+import Foundation
```

Comment:
```suggestion
```

Reply:
Why we should delete it Ahmed?

Reply:
UIKit includes Foundation, so if we included it we then don't have to include Foundation.